Source/Mac/NSImage+Hue.swift

```diff
@@ -12,32 +12,32 @@ class CountedColor {
 
 extension NSImage {
 
-  private func resize(newSize: CGSize) -> NSImage {
+  fileprivate func resize(_ newSize: CGSize) -> NSImage {
```

Comment:
Yeah, that's a burden, you'll get a lot o not defined methods and variable just to know that they have been define private that doesn't mean a file-private anymore!